code/api/views.py

```diff
@@ -231,6 +233,30 @@ def post(self, request, author_id):
 class PostView(View):
 
     def get(self, request, author_id, post_id):
+        author = get_object_or_404(LocalAuthor, pk=int(author_id))
+        post = get_object_or_404(LocalPost, pk=int(post_id))
+        accepted_types = request.headers['Accept']
+
+        if 'image' in accepted_types and post.is_image_post() and post.unlisted:
```

Comment:
Is `'image' in accepted_types` necessary here? I feel like it might cause issues and be overly picky when responding to requests (thinking about other groups here).

Also, I had some issues finding the URL for the image I just created. Could you create some sort of print to console that says the ID of the unlisted post when created? We would ideally want some UI for this to tell the user where to find their unlisted post, but that can be left for another PR.

I also think it would be best to put this logic on a different route. What happens if I go to this URL in the browser. My intuition says I should get the post JSON object back since I just made a GET request. But then how do I see the rendered image if that's what I wanna do? Which will be given preference for a generic GET request at this URL (the JSON post object or the image)?

Reply:
I agree with making a new route. This endpoint wasn't implemented. What I had in mind was to re-use the post endpoint for returning the image if the "image/webp" mime-type was specified. But, I was thinking of giving priority to text/html and application/json later on. I made a new endpoint for a post's image.

Michelle told me she was working on a template for showing unlisted post. I asked her if she could add a button or some obvious way for the user to copy the route to the post's image to their clipboard. The built-in Clipboard api looks promising but I haven't had the chance to look further into it.